sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -498,6 +499,7 @@ export type FileAbortCopyResponse = FileAbortCopyHeaders & {
 
 // @public (undocumented)
 export interface FileAndDirectoryCreateCommonOptions {
+    changedOn?: Date | TimeNowType;
```

Comment:
Can we align the name between "changeTime" and "changeOn". Are they refer to same thing?

Reply:
Updated to changeTime